src/main/java/Wolox/training/models/User.java

```diff
@@ -0,0 +1,106 @@
+package Wolox.training.models;
+
+import Wolox.training.exceptions.BookAlreadyOwnedException;
+
+import javax.persistence.*;
+import java.util.Collections;
+import java.util.List;
+
+@Entity(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private String birthDate;
+
+    @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
[Hibernate has many cascade types](https://www.baeldung.com/jpa-cascade-types) and depending on the case, we should use only the ones we need, because using `CascadeType.ALL` we can get undesired behavior (such as remove additional data). In our case, it is enough to use the **merge** and **refresh** type.
```suggestion
    @ManyToMany(cascade = {CascadeType.MERGE, CascadeType.REFRESH})
```